src/pools/cluster/fixed.ts

```diff
@@ -2,6 +2,7 @@ import { fork, isMaster, setupMaster, Worker } from 'cluster'
 import type { JSONValue, MessageValue } from '../../utility-types'
 import type { PoolOptions } from '../abstract-pool'
 import { AbstractPool } from '../abstract-pool'
+import { IPool } from '../pool'
```

Comment:
We should to add `type` here
```suggestion
import type { IPool } from '../pool'
```